libcaf_net/test/detail/rfc6455.cpp

```diff
@@ -18,193 +16,191 @@
 
 using namespace caf;
 
-namespace {
-
-struct fixture {
-  using impl = detail::rfc6455;
+using impl = detail::rfc6455;
 
-  auto bytes(std::initializer_list<uint8_t> xs) {
-    byte_buffer result;
-    for (auto x : xs)
-      result.emplace_back(static_cast<std::byte>(x));
-    return result;
-  }
-
-  template <class T>
-  auto take(const T& xs, size_t num_bytes) {
-    auto n = std::min(xs.size(), num_bytes);
-    return std::vector<typename T::value_type>{xs.begin(), xs.begin() + n};
-  }
-};
+SUITE("detail.rfc6455") {
 
-} // namespace
+auto bytes(std::initializer_list<uint8_t> xs) {
+  byte_buffer result;
+  for (auto x : xs)
+    result.emplace_back(static_cast<std::byte>(x));
+  return result;
+}
 
-BEGIN_FIXTURE_SCOPE(fixture)
+template <class T>
+auto take(const T& xs, size_t num_bytes) {
+  auto n = std::min(xs.size(), num_bytes);
+  return std::vector<typename T::value_type>{xs.begin(), xs.begin() + n};
+}
```

Comment:
Should we add an anonymous namespace here?

Reply:
No. 🙂

`SUITE("detail.rfc6455")` already makes sure that everything in the suite has internal linkage.